articles/uploading-images-to-cloudinary-from-django-application/index.md

```diff
@@ -0,0 +1,270 @@
+Sometimes while working on applications that require the use of images, a simple process like uploading images to a server can become difficult.
+If you deploy your Django to platform like heroku, you can't save image.
+
+In this tutorial, we look at how to save images in Django using CLoudinary.
+
+Cloudinary is an end-to-end image- and video-management solution for websites and mobile apps, covering everything from image and video uploads, storage, manipulations, optimizations to delivery.
+
+### Prerequisites
+
+To follow along with this tutorial, you need Python3 installed on your machine.
+
+### Creating and setting up a new Django project
+
+Let's start by creating a new virtual environment
+
+```bash
+$ python -m virtualenv env
+
+```
+
+Activate virtual environment
+
+```bash
+$ source virtual/bin/activate
+
+```
+
+Install Django
+
+```bash
+$ pip install django
+
+```
+
+Create new Django project
+
+```bash
+$ django-admin startproject photoapp
+
+```
+
+Create Django app
+
+```bash
+$ django-admin startapp photos
+
+```
+
+Lets add our app to the list of installed apps. Navigate to the "photoapp" directory and edit the settings.py file
```

Comment:
```suggestion
Let's add our app to the list of installed apps. Navigate to the `photoapp` directory and edit the `settings.py` file.
```

Reply:
Always enclose unique words or variables or filenames within ` `` ` to highlight them